src/QMCDrivers/WalkerControlBase.cpp

```diff
@@ -443,23 +440,40 @@ int WalkerControlBase::sortWalkers(MCWalkerConfiguration& W)
 
 int WalkerControlBase::copyWalkers(MCWalkerConfiguration& W)
```

Comment:
Description of algorithm needed in doxygen comment

Reply:
Please add a note to say that this algorithm attempts to minimize large allocations/deallocations. This was in the PR. The new comments say "what" but not "why".